app/assets/stylesheets/components/common/_travel_insurance_wizard_form.scss

```diff
@@ -0,0 +1,20 @@
+.form.js-wizard{
+  position: relative;
+  overflow: hidden;
+
+  .wizard-form-item{
+    transform: translateX(100%);
+    padding: 1px;
+    background-color: #fff;
```

Comment:
We prefer to use a variable `$color-white` here in place of the value `#fff`. 
But in addition is the colour specification necessary here?

Reply:
Removed, it was not required anymore. I was using it when sliding in the element so there wasn't text on text